site-packages/integralstor_gridcell/cifs.py

```diff
@@ -66,6 +66,16 @@ def _generate_local_share(f):
         f.write("  kernel share modes = no\n")
         f.write("  force group = integralstor\n")
         f.write("  browseable = no\n")
+
+        ret, err = cifs_utils.generate_share_header(
+            f, 'integralstor_system_logs')
```

Comment:
We may want to use a shorter share name otherwise testparm gives the following warning : 

WARNING: You have some share names that are longer than 12 characters.
These may not be accessible to some older clients.
(Eg. Windows9x, WindowsMe, and smbclient prior to Samba 3.0.)

Reply:
I saw the warning too but there is another share by the name `integralstor_local_storage` so I presumed that you had previously decided to ignore the warning.